src/venia-pwa-concept/setup/index.md

```diff
@@ -1,3 +1,106 @@
 ---
 title: Setup
----
\ No newline at end of file
+---
+
+Follow the instructions on this page to setup the [Venia PWA concept theme] for Magento 2.
+
+## Prerequisites
+
+* A local development instance of Magento 2.3 or above.
+* [Node Package Manager] (NPM)
+* [NodeJS 8.x LTS]
+
+## Clone repository
+
+Clone the [Venia PWA concept theme] repository into your development environment. 
+
+``` sh
+git clone git@github.com:magento-research/venia-pwa-concept.git
+```
+
+**Note:**
+*For this tutorial, the project is located in the `/Users/magedev/venia-pwa-concept` directory.*
+
+### Vagrant Box instructions
+
+If you are using a virtual machine, make sure it is able to access the new project directory. 
+
+For example, if you are using the [Vagrant Box for Magento 2 developers], use the following steps to add a synced folder to the virtual machine:
+
+1. In the Vagrant box project directory, open the `Vagrantfile` and locate the following line:
+   ``` 
+   config.vm.synced_folder '.', '/vagrant', disabled: true
+   ```
+1. Add an entry similar to the following entry above this line:
+   ```
+   config.vm.synced_folder '/Users/magedev/venia-pwa-concept', '/Users/magedev/venia-pwa-concept', type: "nfs", create: true
```

Comment:
Another point could be added to so that _Vagrant Box for Magento 2 developers_ uses Magento 2.3
```
Copy `etc/config.yaml.dist` as `etc/config.yaml` and update it to use Magento v2.3-develop and PHP7.1  by changing:<br />
`ce: "git@github.com:magento/magento2.git"` to 
`ce: "git@github.com:magento/magento2.git::2.3-develop"`.<br />
And by changing `php_version: "7.0"` to `php_version: "7.1"`
```

Reply:
Thanks for the instructions @rossmc ! I'll make sure to include this in my draft update.